### PR TITLE
Added case insensitive query to login and resolved issue with autoMig…

### PR DESCRIPTION
**Pull request by devan-huapaya**
…rating empty documents

### CODE REVIEW

File: packages/api/src/components/Migrator.ts [hunk 1]
```diff
@@ -119,7 +119,10 @@ export class Migrator {
 					}
 				}
 			}
-			await this.db.collection(collectionName).insertMany(docs)
```
Comment (darthtrevino) [inline packages/api/src/components/Migrator.ts:122]: the create_mock_data script should be run before the migrator pops off

**Review by darthtrevino — APPROVED** [review]
This is fine to me, but for local dev you'll want to run `yarn create_mock_data`. Since we do some asset processing in the webapp too, we can probably sequence this across the repo as a pre-build step called `assets`.

File: packages/api/src/components/Migrator.ts (hunk 1 above)
Comment (devan-huapaya) [reply packages/api/src/components/Migrator.ts:122]: Yes, I was hesitant to include this but if the mock data is set to create 0 docs for any collection. (contacts and engagements in my case) it stops running the script

Comment (devan-huapaya) [reply packages/api/src/components/Migrator.ts:122]: Yep! I had my create_mock_data set up to not create contacts and engagements and having 0 docs in a collection stopped the entire script

**Pull request merged**